EnglishFilter/sections/foreign.txt

```diff
@@ -2350,6 +2350,8 @@ mob.hr##.side_banner
 !------------------------------ Czech and Slovakian ------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/77726
+kosicednes.sk#?#.related-article:contains(reklamná správa)
```

Comment:
better `kosicednes.sk##[data-banner-type]` (without tag - for `DIV` and `A`)

Reply:
Okay.